sopel/tools/__init__.py

```diff
@@ -199,6 +198,13 @@ def deprecated_func(*args, **kwargs):
     return deprecated_func
 
 
+# This has to be *after* the definition of `deprecated()` or we
+# get "AttributeError: partially initialized module 'sopel.tools' has no
+# attribute 'deprecated' (most likely due to a circular import)" when trying
+# to use the decorator in submodules.
+from . import time, web  # NOQA
```

Comment:
See #2206 for future PR.

Reply:
That's not blocking this, right? I figured you'd keep moving stuff around to avoid circular imports and other dependency-related code smells as you have in the past (e.g. #2179).

Reply:
It isn't! I just wanted to comment with a "TODO" and a link to the issue I opened to track that. :)